### DISCUSSION TITLE
Why do I need print() on Jupyter ?

### DISCUSSION THREAD

**Discussion by danielreispereira** [h2oai/datatable#3496] [Q&A]
Is this by design? Do I need to use print() around a datatable on Python to see the datatable on console/jupyter? 
Is there a workaround to get the datatable printed on terminal when its name is run ?

![image](https://github.com/h2oai/datatable/assets/4158858/576608bd-0df6-43ff-be51-82c38d360897)

Thanks

**Comment by samukweku** [comment]
@danielreispereira Have a look at your version. I am on 1.1 and it prints fine without the `print()` function : 

```
In [82]:  from datatable import dt

In [83]: frame = dt.Frame([1])

In [84]: frame
Out[84]:
   |   C0
   | int8
-- + ----
 0 |    1
[1 row x 1 column]

In [85]: dt.__version__
Out[85]: '1.1.0a0+build.1695251213.samueloranyeli'
```
